test_sim_volumes.py

```diff
@@ -0,0 +1,56 @@
+"""Test sim_volumes."""
+import os
+
+import numpy as np
+
+import sim_volumes
+
+os.environ["KMP_DUPLICATE_LIB_OK"] = "TRUE"
+
+
+class TestSimVolumes:
+    @staticmethod
+    def test_get_random_quat():
+        quaternions = sim_volumes.get_random_quat(5)
+        assert quaternions.shape == (5, 4)
+        assert type(quaternions) is np.ndarray
+
+    @staticmethod
+    def test_uniform_rotations():
+        rot, quat = sim_volumes.uniform_rotations(5)
+        assert rot.shape == (5, 3, 3)
+        assert quat.shape == (5, 4)
+
+    @staticmethod
+    def test_modify_weight():
+        points = np.asarray([[1, 0, 0, 3], [0, 8, 0, 5], [0, 0, 7, 9]])
+        vol_size = 64
+        center = 2
+        volume = np.zeros((vol_size,) * 3)
+        volume = sim_volumes.modify_weight(points, volume, vol_size, center)
+        assert volume.shape == (vol_size,) * 3
+        assert volume[1][1][1] == 3.4866983294215885e-164
```

Comment:
How did you get this value?

If you run `sim_volumes.modify_weight` on your computer and then copy-pasted it here, it's slightly less effective as a unit-test (it's still useful, as it protects your code from being modified by other contributors).

Is there a simpler test (with a simpler volume) that could be run here, so that the volume[1][1][1] can be computed manually - so that the manual computation is the check for the `modify_weight`'s computation?